app/views/books/new.html.erb

```diff
@@ -1,17 +1,18 @@
 <h1>Add new book</h1>
 
-<%= form_tag :action => 'create' do %>
-  <p><label for = "book_title">Title</label>:
-    <%= text_field 'books', 'title' %></p>
+<%= form_with model: @book do |f| %>
+  <p><%= f.label :title %>:
+    <%= f.text_field :title %></p>
 
-  <p><label for = "author">Author</label>:
-    <%= text_field 'books', 'author' %></p>
+  <p><%= f.label :author %>:
+    <%= f.text_field :author %></p>
 
-  <p><label for = "isbn">ISBN</label>:
-    <%= text_field 'books', 'isbn' %></p>
-
-  <%= submit_tag "Create" %>
+  <p><%= f.label :isbn %>:
+    <%= f.text_field :isbn %></p>
 
+  <%= f.submit "Create" %>
 <% end -%>

```

Comment:
Just a note that it might be worth extracting your add and edit forms into a shared partial view (https://guides.rubyonrails.org/action_view_overview.html#partials)